### DISCUSSION TITLE
Questions regarding of obtaining data to draw overlaid figures of dimensionaless kratky plot

### DISCUSSION THREAD

**Discussion by nanolife1** [biosaxs-com/atsas-community#1943] [Primary data processing] [answered]
Dear Biosaxs,
I am trying to draw a overlaid figures of two dimensionless Kratky plots but I don't know how to export data of each dimensionaless Kratky plot. I appreciate if you can give me some instructions.
Thank you very much.
Na

**Comment by franke-biosaxs** [comment, marked as answer]
In [ATSAS-4.0](https://github.com/biosaxs-com/atsas-community/releases), you can right click the plot(s) you are interested in and select "Export to CSV" in the menu. Then import both datasets in your favourite plotting application and create the final plot. 

Column order: s, I1, err1, I2, err2, ...

**Comment by nanolife1** [comment]
I will do that. Thank you so much.
Na
